chat.php

```diff
@@ -197,13 +197,13 @@ function chat() {
 			echo "
 				<tr>
 					<td style='text-align:center;'>
-					<div id='user_data_container' style='display: flex;flex-direction:row'>
+					<div id='user_data_container'>
```

Comment:
I think this is going to break it, why do we need to change this?

Reply:
Oh yeah sorry, I was testing this on the mobile view. Don't change this inline. mb

Reply:
Gotcha - You can feel free to migrate inline styles into the stylesheets so you can control per-layout, but need to add it to all the existing stylesheets if so. Should we just revert all these changes right here for now?

Reply:
```suggestion
					<div id='user_data_container' style='display: flex;flex-direction:row'>
```